storage/src/vespa/storageframework/generic/clock/time.h

```diff
@@ -170,4 +166,39 @@ inline MicroSecTime MilliSecTime::getMicros() const {
     return MicroSecTime(getTime() * 1000);
 }
 
+inline MicroSecTime
+operator + (MicroSecTime a, MicroSecTime b) {
+    MicroSecTime result(a);
+    result += b;
+    return result;
+}
+
+inline MilliSecTime
+operator + (MilliSecTime a, MilliSecTime b) {
+    MilliSecTime result(a);
+    result += b;
+    return result;
+}
+
+inline SecondTime
+operator + (SecondTime a, SecondTime b) {
+    SecondTime result(a);
+    result += b;
+    return result;
+}
+
+inline MicroSecTime
+operator - (MicroSecTime a, MicroSecTime b) {
+    MicroSecTime result(a);
+    result -= b;
+    return result;
+}
+
+inline SecondTime
+operator - (SecondTime a, SecondTime b) {
+    SecondTime result(a);
+    result -= b;
+    return result;
+}
```

Comment:
These could probably be generalized as a single overload per operator, using the "micros per unit" template arg to properly normalize values. But that's basically what `std::chrono` already does, so no point if we're already replacing with that.

Reply:
That happens further down the PR pipeline.